src/gui/systray.cpp

```diff
@@ -130,6 +149,37 @@ void Systray::slotNewUserSelected()
     UserAppsModel::instance()->buildAppList();
 }
 
+void Systray::slotUnpauseAllFolders()
+{
+    setPauseOnAllFoldersHelper(false);
+}
+
+void Systray::slotPauseAllFolders()
+{
+    setPauseOnAllFoldersHelper(true);
+}
+
+void Systray::setPauseOnAllFoldersHelper(bool pause)
+{
+    const auto accounts = [=] {
```

Comment:
The purpose of this is to un-smart the pointer, right? I assume you have a good reason, might be worth a comment though

Reply:
Honestly it's just older code which I pimped up a bit, I dunno why they un-smart-ed the pointer back then. I'll see to add a comment or remove that step.

Reply:
Went for the comment, a proper fix would be more impactful than I'd like in this PR.